py3status/modules/weather_yahoo.py

```diff
@@ -81,9 +81,13 @@ class Py3status:
     units = 'c'
     woeid = None
 
+    def post_config_hook(self):
+        if not self.woeid:
+            raise Exception('not configured')
```

Comment:
`woeid not configured` feels more informative.

Reply:
I'd like to keep it generic.... so we can keep everything a bit more consistent. 

Right now, there is a `__init__(self)` in `battery_level.py`.... Should I move it to `post_config_hook()` or leave it there? That one, I'm not so sure. Can't be sure if it's an accident...

Reply:
* `window_manipulation: format not configured` (formal)
* `window_manipulation: missing format`. (simple)

I like the second one and updated this PR too. Also, we missed them by not paying attention to anything or just starting a module... rather than not configuring anything on purpose. This, I think, is more friendly too.